tests/llm/utils/mock_toolset.py

```diff
@@ -62,15 +63,24 @@ def _auto_generate_mock_file(self, params: Dict):
 
         logging.info(f"Invoking tool {self.unmocked_tool}")
         output = self.unmocked_tool.invoke(params)
-        output = strip_ansi(output)
```

Comment:
`ToolMock` defined in this file likely needs to change so that `return_value` is the structured output.